### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by YuriyBilskiy**
[DEMO LINK](https://yuriybilskiy.github.io/react_static-list-of-todos-js/)

### CODE REVIEW

File: src/components/TodoInfo/TodoInfo.jsx [hunk 1]
```diff
@@ -1,4 +1,13 @@
...
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const { title, completed, user } = todo;
+
+  return (
+    <article className={`TodoInfo TodoInfo--${completed ? 'completed' : ''}`}>
```
Comment (VitaliyBodnar) [inline src/components/TodoInfo/TodoInfo.jsx:7]: it's better to use classname library for making dynamic classname

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

File: src/components/TodoInfo/TodoInfo.jsx [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+  const { title, completed, user } = todo;
+
+  return (
+    <article className={cn('TodoInfo', {
+      'TodoInfo--completed': completed,
+    })}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+
+      <UserInfo user={user} />
```
Comment (DarkMistyRoom) [inline src/components/TodoInfo/TodoInfo.jsx:14]: don't render UserInfo if the is no user

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Good! One fix is needed

**Review by IvanFesenko — APPROVED** [review]
well done

**Pull request closed without merging**